spec/services/preview_service_spec.rb

```diff
@@ -0,0 +1,117 @@
+# frozen_string_literal: true
+
+RSpec.describe PreviewService do
+  before do
+    stub_any_publishing_api_put_content
+    stub_any_asset_manager_call
+  end
+
+  describe "#create_preview" do
+    it "updates the Publishing API" do
+      edition = create(:edition)
+      request = stub_publishing_api_put_content(edition.content_id, {})
+      PreviewService.new(edition).create_preview
+      expect(request).to have_been_requested
+    end
+
+    it "marks the edition as revision synced" do
```

Comment:
Looking at the test for the new service, how about 'revision_synced'?

Reply:
I'm not quite sure what you mean here, do you mean `it "marks the edition as 'revision_synced'"`?